gammapy/utils/testing.py

```diff
@@ -15,7 +17,21 @@ def assert_quantity(actual, desired, *args, **kwargs):
     if not isinstance(actual, Quantity):
         raise ValueError("actual must be a Quantity object.")
     if not isinstance(desired, Quantity):
-        raise ValueError("actual must be a Quantity object.")
+        raise ValueError("desired must be a Quantity object.")
+
+    assert actual.unit == desired.unit
+    assert_allclose(actual, desired, *args, **kwargs)
+
+
+def assert_angle(actual, desired, *args, **kwargs):
```

Comment:
I don't think this is really useful in addition to the quantity assert helper function?